instance/models/openedx_instance.py

```diff
@@ -191,12 +192,16 @@ def first_activated(self):
         except models.ObjectDoesNotExist:
             return
 
-    @log_exception
-    def spawn_appserver(self):
+    def _spawn_appserver(self):
         """
-        Provision a new AppServer.
+            Provision a new AppServer
+
+            Optionally mark the new AppServer as active when the provisioning completes.
+            Optionally retry up to 'num_attempts' times.
+            Optionally tag the instance with 'success_tag' when the deployment succeeds,
+            or failure_tag if it fails.
```

Comment:
I don't see any of these options in this function. Perhaps this documentation is intended for ``spawn_appserver``?